meta-lxatac-software/conf/distro/tacos.conf

```diff
@@ -0,0 +1,50 @@
+DISTRO = "tacos"
+DISTRO_NAME = "TAC OS - The LXA TAC operating system"
+DISTRO_VERSION = "23.11+dev"
+DISTRO_CODENAME = "tacos-mickledore"
+
+MAINTAINER = "Linux Automation GmbH <info@linux-automation.com>"
+
+LOCALCONF_VERSION = "2"
+
+DISTRO_FEATURES = "argp ext2 largefile usbgadget usbhost wifi xattr nfs zeroconf multiarch systemd usrmerge"
+
+# Select systemd as init manager
+INIT_MANAGER ?= "systemd"
+
+DISTRO_EXTRA_RDEPENDS += "packagegroup-core-boot ptx-profile"
+DISTRO_EXTRA_RRECOMMENDS += "kernel-module-af-packet"
```

Comment:
These two lines could be omitted I guess. They were copied from poky.conf to ptx.conf once but are not really relevant to the system (or explicitly added anyway).

Reply:
I've removed the two lines now. Starting with as little cargo-culting as possible sounds good to me.